sdk/framework/openapi.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/hashicorp/vault/sdk/logical"
 	"github.com/hashicorp/vault/sdk/version"
 	"github.com/mitchellh/mapstructure"
+	"golang.org/x/text/cases"
```

Comment:
currently an [indirect](https://github.com/hashicorp/vault/blob/1440b0bfc7ef7518240926909d3a97c2d4a27e9c/go.mod#LL423), would need to update go.mod

Reply:
`go mod tidy` seems to be happy with it